airbyte-config/config-models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -55,3 +55,6 @@ properties:
     format: date
   resourceRequirements:
     "$ref": ActorDefinitionResourceRequirements.yaml
+  protocol_version:
```

Comment:
just passing by, seems like we're using `camelCase` for most of these - should it be `protocolVersion` here?

Reply:
Good question, I think I saw a few [comments](https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L365) about prefering snake_case over camelCase for the attributes in the models.
The generated output remains camelCase for the Java objects.
No strong opinion here.

Reply:
Ah yeah - main reason why I pointed this out is since at least specifically within these StandardXYZDefinitions schemas it seems like camel case is being used

Reply:
Same conclusion, change on the way